src/sql/azdata.d.ts

```diff
@@ -3094,6 +3094,7 @@ declare module 'azdata' {
 		 * means the event will propagate up to any parents that have handlers (such as validate on Dialogs)
 		 */
 		stopEnterPropagation?: boolean;
+		validationErrorMessage?: string;
```

Comment:
Add this to azdata.proposed.d.ts instead

Reply:
And make sure you're updating the required ADS version for the extension as well

---
In reply to: [369685962](https://github.com/microsoft/azuredatastudio/pull/8909#discussion_r369685962) [](ancestors = 369685962)